README.md

```diff
@@ -1,46 +1,94 @@
 # ethminer
 
+[![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg)](https://github.com/RichardLitt/standard-readme)
+[![Gitter](https://img.shields.io/gitter/room/nwjs/nw.js.svg)][Gitter]
+[![Releases](https://img.shields.io/github/downloads/atom/atom/total.svg)][Releases]
+
+> Ethereum miner with OpenCL, CUDA and stratum support
+
 The ethminer is an Ethereum GPU mining worker. It origins in [cpp-ethereum]
 project (where GPU mining has been discontinued). Then hugely improved in
 [Genoil's fork].
 
 ### Features
 
+- OpenCL mining
 - Nvidia CUDA mining
 - realistic benchmarking against arbitrary epoch/DAG/blocknumber
 - on-GPU DAG generation (no more DAG files on disk)
 - stratum mining without proxy
 - OpenCL devices picking
 - farm failover (getwork + stratum)
 
-## Community
 
-[![Gitter](https://img.shields.io/gitter/room/nwjs/nw.js.svg)][Gitter]
+## Table of Contents
+
+- [Install](#install)
+  - [Building from source](#building-from-source)
+  - [CMake build options](#cmake-build-options)
+- [Usage](#usage)
+- [Contribute](#contribute)
+- [F.A.Q.](#faq)
+
+
+## Install
+
+[![Releases](https://img.shields.io/github/downloads/atom/atom/total.svg)][Releases]
+
+Standalone executables for Linux, macOS and Windows are provided in
+[the Releases section][Releases].
+Download an archive for your operating system and unpack the content to a place
+accessible from command line. The ethminer is ready to go.
+
+### Building from source
+
+This project uses [CMake] and [Hunter] package manager.
+
+```sh
+mkdir build; cd build
+cmake ..
+cmake --build .
+```
```

Comment:
This is linux specific but maybe add a `make install` at the end so the binary gets copied to `/usr/local/bin`